src/main/java/com/google/api/codegen/transformer/csharp/CSharpApiMethodTransformer.java

```diff
@@ -226,24 +207,15 @@ protected void setServiceResponseTypeName(
         // Paged streaming methods.
         if (methodConfig.isFlattening()) {
           for (FlatteningConfig flatteningGroup : methodConfig.getFlatteningConfigs()) {
-            GapicMethodContext methodContext =
+            MethodContext methodContext =
                 context.asFlattenedMethodContext(requestMethodContext, flatteningGroup);
-            apiMethods.add(
```

Comment:
This code block has been moved, unchanged, to the body of `generatePageStreamingFlattenedMethods()`